spec/models/user_spec.rb

```diff
@@ -12,16 +15,21 @@
     expect(user).to_not be_valid
     expect(user.errors[:password_confirmation]).to_not be_blank
   end
-end
 
-RSpec.describe 'Confirm admin role' do
-  it 'is not an admin if the role is not admin' do
-    user = FactoryGirl.create(:user)
-    expect(user.admin?).to eq(false)
+  describe 'Confirm username' do
+    it 'should have a username' do
+      expect(user.username).to eq(username)
```

Comment:
I'm not feeling like its super useful as it stands. It's not particularly hurting anything either, so It's fine for the sake of getting the branch merged in